api/models/Dependency.js

```diff
@@ -38,6 +38,9 @@ module.exports = {
 
           return data;
         });
+      },
+      findByDependantForIndependentDownloads: function(package){
```

Comment:
database name should not be in the queries, the name is not the same on the production server